curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65c63fd9b387f0a6c5bf6a72.md

```diff
@@ -1,13 +1,13 @@
 ---
-id: 655a167ea3e96512bf1343ce
-title: Step 20
+id: 65c63fd9b387f0a6c5bf6a72
+title: Step 24
 challengeType: 20
-dashedName: step-20
+dashedName: step-24
 ---
 
 # --description--
 
-Inside the `search` method, delete `pass` and call the private helper method `_search` with the following arguments. You will define `_search` in the next steps.
+Inside the `search` method, delete `pass` and call the helper method `_search` with the following arguments.
 
 - `self.root`: This is the root of the binary search tree. The search starts from the root.
 - `key`: This is the value that the user wants to find in the binary search tree.
```

Comment:
Not able to comment in the right place. Anyway, I would remove that "private" from below, since the `_` does not make the method really private. "helper" seems more appropriate and it has been already used before.